sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -40,10 +42,13 @@ class GestureArena {
 
   void _resolve(Object key, GestureArenaMember member, GestureDisposition disposition) {
     List<GestureArenaMember> members = _arenas[key];
+    if (members == null && disposition == GestureDisposition.accepted)
```

Comment:
Do you want to support a trivial gesture that always accepts or always rejects? If you do, you will want to handle the case here of a rejected gesture asking to be accepted or rejected after the list is empty.